src/Features/CSharp/Portable/SignatureHelp/AbstractCSharpSignatureHelpProvider.cs

```diff
@@ -106,9 +106,9 @@ protected IList<TaggedText> GetAwaitableUsage(IMethodSymbol method, SemanticMode
         }
 
         /// <summary>
-        /// If an argument name does not correspond to a parameter name, this method is inacceptable. We'll filter it out.
+        /// If an argument name does not correspond to a parameter name, this method is unacceptable. We'll filter it out.
         /// </summary>
-        protected static bool IsInacceptable(SeparatedSyntaxList<ArgumentSyntax> arguments, IMethodSymbol method)
+        protected static bool IsUnacceptable(SeparatedSyntaxList<ArgumentSyntax> arguments, IMethodSymbol method)
```

Comment:
instead of having `IsUnacceptable` which tehn has to eb used like `=> !IsUnacceptable`, consider just having `=> IsAcceptable`.

Reply:
this cmment still applies, unless you strongly like the negative form here.